commit fddf13191b475f9d1a09b0d6a183ac7ca73d444f
Author: Ron Arts <ron.arts@novantum.com>
Date:   Mon Feb 12 17:53:37 2018 +0100

    react-native-web aliases `import ... from 'react-native'` into `import ... from 'react-native-web'`.
    But it does not implement undocumented or iOS/Android-only components, not even stubs for it.
    So I added a stub in this package, which provides a component which just sais not implemented.:wq

diff --git a/src/TabViewPagerAndroid.web.js b/src/TabViewPagerAndroid.web.js
new file mode 100644
index 0000000..738873d
--- /dev/null
+++ b/src/TabViewPagerAndroid.web.js
@@ -0,0 +1,6 @@
+import { Text } from 'react-native';
+
+export default TabViewPagerAndroid = () => (
+  <Text>The TabViewPagerAndroid is not supported on React Native Web</Text>
+)
+
